srcpkgs/ibus-chewing/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-chewing'
+pkgname=ibus-chewing
+version=1.5.1
+revision=1
+archs="x86_64"
```

Comment:
Why is this restricted to x86_64? (if, for example, it doesn't cross compile, it should instead be marked as nocross)

Reply:
This was not resolved, please don't mark it resolved